### ISSUE TITLE
Update site is not available anymore

### ISSUE THREAD

**Issue by philippecade** [rombert/ereviewboard#159]
The update site as listed below is not available anymore and gives a 404 error.

http://rombert.github.io/ereviewboard/update/

**Comment by rombert** [comment]
Hi @philippecade  and thanks for the report. The p2 update site loads fine for me in Eclipse. 

![image](https://user-images.githubusercontent.com/21237/66990033-b953a100-f0c5-11e9-909d-0d7a706261bf.png)

Did you just navigate using the browser or did you try to install from Eclipse?

**Comment by philippecade** [comment]
We are installing ereviewboard via Eclipse Oomph and I got an error that this update site was not available.

Then I tried in the browser and got the 404. Thinking about it a bit more, the 404 probably comes from the fact that the update site doesn't have an index.html file.

Anyway it looks like it is working again now when using Oomph, so I'll close this issue. Might have been a hitch with the GitHub pages.
